tests/simple.rs

```diff
@@ -135,3 +135,31 @@ fn test_display_should_fail() {
         "Invalid input string: cannot be parsed as a relative time"
     );
 }
+
+#[test]
+fn test_from_str_at_date_day() {
+    let today = OffsetDateTime::now_utc().date();
+    let yesterday = today - Duration::days(1);
+    assert_eq!(
+        from_str_at_date(yesterday, "2 days").unwrap(),
+        Duration::days(1)
+    );
+}
+
+#[test]
+fn test_invalid_input_at_date() {
+    let today = OffsetDateTime::now_utc().date();
+    let yesterday = today - Duration::days(1);
+    let result = from_str_at_date(yesterday, "foobar");
+    println!("{result:?}");
+    match result {
+        Err(ParseDurationError::InvalidInput) => assert!(true),
+        _ => assert!(false),
+    }
+
+    let result = from_str_at_date(yesterday, "invalid 1r");
```

Comment:
I think the `yesterday` var is noise because it is irrelevant for the test what date it is, it only has to be a valid `Date` object.
```suggestion
    let result = from_str_at_date(today, "foobar");
    println!("{result:?}");
    match result {
        Err(ParseDurationError::InvalidInput) => assert!(true),
        _ => assert!(false),
    }

    let result = from_str_at_date(today, "invalid 1r");
```